curriculum/challenges/english/07-scientific-computing-with-python/learn-lambda-functions-by-building-an-expense-tracker/6669539c1379793f9cb8917c.md

```diff
@@ -0,0 +1,61 @@
+---
+id: 6669539c1379793f9cb8917c
+title: Step 5
+challengeType: 20
+dashedName: step-5
+---
+
+# --description--
+
+An other thing you can do with bracket notation is change an element value.
+
```

Comment:
It is evident now, but maybe we should say that lists can hold different data types.